src/rgw/rgw_civetweb.cc

```diff
@@ -112,6 +112,7 @@ void RGWCivetWeb::init_env(CephContext *cct)
 
   env.set("REMOTE_ADDR", info->remote_addr);
   env.set("REQUEST_METHOD", info->request_method);
+  env.set("HTTP_VERSION", info->http_version);
```

Comment:
could you please make a similar change to `ClientIO::init_env()` in rgw_asio_client.cc?

Reply:
For the upcoming new frontend？ Yes！